lua/ulx/modules/sh/cfc_trainfuck.lua

```diff
@@ -0,0 +1,29 @@
+CATEGORY_NAME = "Fun"
+
+local function trainFuck( callingPlayer, targetPlayers )
+    for _, ply in pairs( targetPlayers ) do
+        local train = ents.Create( "prop_physics" )
+        train:SetModel( "models/props_trainstation/train001.mdl" )
+        train:SetPos( ply:GetPos() + Vector( 0, -1000, 300 ) )
+        train:Spawn()
+
+        local phys = train:GetPhysicsObject()
+
+        if IsValid( phys ) then
+            if ply:GetMoveType( MOVETYPE_NOCLIP ) then
+                ply:SetMoveType( MOVETYPE_WALK )
+            end
+            ply:ExitVehicle()
+            phys:ApplyForceCenter( ( ply:GetPos() - train:GetPos() ) * 1000000000 )
```

Comment:
```suggestion

            phys:ApplyForceCenter( ( ply:GetPos() - train:GetPos() ) * 1000000000 )

```
Bit of spacing here would be nice, to "categorise" what the chunks of code are doing